analytical/templatetags/heap.py

```diff
@@ -0,0 +1,54 @@
+"""
+Heap template tags and filters.
+"""
+
+import re
+
+from django.template import Library, Node, TemplateSyntaxError
+
+from analytical.utils import is_internal_ip, disable_html, get_required_setting
+
+
+HEAP_TRACKER_ID_RE = re.compile(r'^\d+$')
+TRACKING_CODE = """
+    <script type="text/javascript">
+  window.heap=window.heap||[],heap.load=function(e,t){window.heap.appid=e,window.heap.config=t=t||{};var r=document.createElement("script");r.type="text/javascript",r.async=!0,r.src="https://cdn.heapanalytics.com/js/heap-"+e+".js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(r,a);for(var n=function(e){return function(){heap.push([e].concat(Array.prototype.slice.call(arguments,0)))}},p=["addEventProperties","addUserProperties","clearEventProperties","identify","resetIdentity","removeEventProperty","setEventProperties","track","unsetEventProperty"],o=0;o<p.length;o++)heap[p[o]]=n(p[o])};
```

Comment:
Flake8 complains that this line is too long. Can you split this up somehow w/o breaking the code?

Reply:
I've taken the quick way out and `#noqa`d the line. It's the Heap supplied code and I didn't fancy trying to unwrap that and be confident it wasn't going to break something at the Heap end. Hope that's okay?